.github/workflows/setupmc.yml

```diff
@@ -0,0 +1,27 @@
+            - name: Setup Minecraft
+  # You may pin to the exact commit or the version.
+  # uses: mcenv/setup-minecraft@2d092242881552b8ee463d9b4687bd38ef58a396
+  uses: mcenv/setup-minecraft@v3.3.2
+  with:
+    # Minecraft version to use
+    version: 1.12.2
+    # Whether to install Minecraft
+    install: # optional, default is true
+    # Whether to cache Minecraft
+    cache: # optional
+    # Number of retries to download Minecraft
+    retries: # optional, default is 3
+
+- uses: actions/checkout@v4
+- id: minecraft
+  uses: mcenv/setup-minecraft@v3
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (bug_risk):** Mismatched version of 'mcenv/setup-minecraft' action.

The version of 'mcenv/setup-minecraft' used here is v3, which is inconsistent with the earlier use of v3.3.2. Ensure version consistency to avoid unexpected behavior.